docs/confluence/advanced/methods.md

```diff
@@ -1,11 +1,228 @@
+
+Internal Function Summaries
+===========================
+
+Summaries Always Inserted
+-------------------------
+
+Summaries for **external** functions are only inserted when an implementation
+of that function cannot be found, and so we default to some summary that we,
+the verifier, sees fit. _However_, internal functions can always be found and
```

Comment:
For clarity and grammar, I would change the beginning of line 24 to "...as the verifier, see fit."